### DISCUSSION TITLE
Alguém para Instalar ou fazer comigo passo-a-passo. Quanto seria?

### DISCUSSION THREAD

**Discussion by maycon1977** [RamonSilva20/mapos#2253] [Q&A]
Olá, pessoal. Tenho uma versão (3.14.2) que fui personalizando ao longo dos anos e uso o sistema online na hostgator.

Quero atualizar o meu sistema pois já colocaram várias melhorias que me serão uteis. Sei que perderei algumas personalizações, mas acredito que após a instalação com esse sistema mais atualizado, consigo fazer as modificações que são essenciais para meu uso via tentativa e erro como fiz ao longo dos anos.

A minha versão não precisou 'rodar o composer'... Era só fazer o banco de dados, subir e configurar... Eu mesmo fiz na época, tinha tempo e disposição para isso. Hoje não tenho nen um e nem outro...  Sei que muitos vão dizer que é fácil, que é só seguir o tutorial, video no youtube, etc, porém já li e reli algumas vezes e sempre me dá receio de mexer no servidor, linhas de comando, etc.

Dito isso...

Preciso de alguém que faça somente a instalação ou que possa perder 1 hora do seu tempo online fazendo um passo-a-passo comigo. Gostaria de saber se alguém consegue fazer isso comigo ou por mim e quanto custaria. Vou colocar meu whatsapp para contato.

(21) 98807-8360 - Maycon

Desde já, agradeço!

**Comment by FlaviusR** [comment]
Olá Maycon. Eu também fiz umas personalizações no meu, adaptado para oficinas mecânicas. 
Estou fazendo uma nova instalação/atualização de um cliente nesse momento.
Mais tarde eu te chamo no WhatsApp para informar se ficou okay e posso te auxiliar ou fazer por você.
Valeu.

**Reply by maycon1977** [reply]
Muito obrigado.

**Comment by willph** [comment]
Para fica fácil pra ti, instala ele no seu computador em local host.

Depois compacta todo o arquivo e sobe para sua hospedagem.

De resto só configurar os dados no config.php e data base.php.

Pelo que eu entende, como você entende um bocadinho, não será difícil
fazer-lo.

Qualquer dúvida tem o pessoal do grupo mapos no WhatsApp que sempre dá uma
força.

Em seg., 23 de out. de 2023 08:59, Maycon Teixeira ***@***.***>
escreveu:

**Comment by KelvinEger** [comment]
> Olá, pessoal. Tenho uma versão (3.14.2) que fui personalizando ao longo dos anos e uso o sistema online na hostgator.
> 
> Quero atualizar o meu sistema pois já colocaram várias melhorias que me serão uteis. Sei que perderei algumas personalizações, mas acredito que após a instalação com esse sistema mais atualizado, consigo fazer as modificações que são essenciais para meu uso via tentativa e erro como fiz ao longo dos anos.
> 
> A minha versão não precisou 'rodar o composer'... Era só fazer o banco de dados, subir e configurar... Eu mesmo fiz na época, tinha tempo e disposição para isso. Hoje não tenho nen um e nem outro... Sei que muitos vão dizer que é fácil, que é só seguir o tutorial, video no youtube, etc, porém já li e reli algumas vezes e sempre me dá receio de mexer no servidor, linhas de comando, etc.
> 
> Dito isso...
> 
> Preciso de alguém que faça somente a instalação ou que possa perder 1 hora do seu tempo online fazendo um passo-a-passo comigo. Gostaria de saber se alguém consegue fazer isso comigo ou por mim e quanto custaria. Vou colocar meu whatsapp para contato.
> 
> (21) 98807-8360 - Maycon
> 
> Desde já, agradeço!

Olá Maycon, eu tenho uma versão personalizada dele, onde já disponibilizo um ambiente pronto para você utilizar. 
Te chamo mais tarde no WhatsApp para que possamos conversar, beleza?
